src/engine/include/GUIDOEngine.h

```diff
@@ -670,6 +670,10 @@ units.
 	*/
 	GUIDOAPI(void) 	GuidoGetDefaultPageFormat( GuidoPageFormat* format );
 
+    /** \brief Gives the default score staves format (one staff at a time).
```

Comment:
The API should be fully documented. The function name is a bit confusing: compared to the above API, it's not the default page format but the score page format, thus I suggest GuidoSetStaffFormat, the parameters are enough to disambiguate the scope.
You should also provide the counterpart call: GuidoGetStaffFormat